.github/workflows/reusable-lint.yml

```
@@ -0,0 +1,19 @@
+name: Reusable Lint Workflow
+
+on: workflow_call
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - uses: actions/setup-node@v2.5.1
+        with:
+          node-version: "16.x"
+          registry-url: "https://registry.npmjs.org"
+      # Check the package.json is correctly formed
+      - run: npx @skypack/package-check
+      # Install dependencies:
+      - run: npm ci
+      # Run the linting command:
+      - run: npm run lint
```

```suggestion
      - run: npm run lint:ci
```

Generally, we would want two different actions by default, one for running the lint and auto-fixing stuff, one for running the lint and not fixing stuff, in CI.

---

We've `npm run lint` and `npm run lint:fix` for that, I think. (though some projects use `npm run lint:check` which I personally find a bit awkward)

---

Haha I think my preference goes `lint:check` > `lint` > `lint:ci`, but I don't have a strong opinion. I'm happy aligning on `lint` and `lint:fix`.